graphrag/config/create_graphrag_config.py

```diff
@@ -480,9 +480,15 @@ def hydrate_parallelization_params(
 
         with (
             reader.use(values.get("local_search")),
-            reader.envvar_prefix(Section.local_search),
+            reader.envvar_prefix(Section.llm),
```

Comment:
why do you reading llm section environment? 
This will drop the original param like community_prop, top_k_entities and etc. 
I think it's not good.

Reply:
Yes I made a comment about that. Currently there is no dedicated variable to set the temperature, top_p, and n parameters for a local/global search call. It should be changed back to local_search when the pr for that is merged.

Reply:
I set it to llm to do local testing, otherwise the parameters would be None and the call will be passed with the default params.